From: SeongJae Park <sj@kernel.org>
Date: Mon, 10 Feb 2025 09:52:36 -0800
Subject: [PATCH] ==== damos_walk fixes ====

mm/damon/core: fix wrong and/or useless damos_walk() behaviors

damos_walk() can finish working earlier or later than expected, and
start earlier than practical.  First two behaviors are clearly wrong
behavior (doesn't follow the documentation) and all three behaviors are
only making the feature useless.  Fix those.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 hkml_cv_bogus/hkml_cv_bogus_ghod0kn5 | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 hkml_cv_bogus/hkml_cv_bogus_ghod0kn5

diff --git a/hkml_cv_bogus/hkml_cv_bogus_ghod0kn5 b/hkml_cv_bogus/hkml_cv_bogus_ghod0kn5
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.39.5

